src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import {Routes,Route} from "react-router";
+import  {Home} from './pages/Home/Home';
+import {MonthStatistics} from "./pages/MonthStatistics/components/MonthStatistics";
+import {Header} from "./pages/shared/Header/Header";
+import {Container} from "./styles/styles"
+
```

Comment:
You can define your routes in separate file "routes.tsx" and import as routes by hook useRoutes
For more check official docs: https://reactrouter.com/docs/en/v6/hooks/use-routes
```suggestion
const element = useRoutes(routes);
```